client/packages/system/src/Stock/Components/Repack/RepackModal.tsx

```diff
@@ -130,6 +146,19 @@ export const RepackModal: FC<RepackModalControlProps> = ({
         />
       }
       cancelButton={<DialogButton variant="cancel" onClick={onClose} />}
+      reportSelector={
+        <ReportSelector context={ReportContext.Repack} onClick={printReport}>
```

Comment:
the disabled prop on the button won't work because the `onClick` is on the parent component and because there is only one report available, the `ReportSelector` is returning a `div` with an `onClick`. 

I think you should add a `disabled` prop to `ReportSelector` and perhaps also rename `onClick` to `onPrint` as that makes it clearer